net-mail/dot-forward/dot-forward-0.71-r4.ebuild

```diff
@@ -46,6 +48,7 @@ src_prepare() {
 }
 
 src_configure() {
+	append-cflags
```

Comment:
Did you mean to put -std=gnu89 or something?

Reply:
Head -> table. 
i actually meant to remove this line altogether.